tools/SourceKit/lib/SwiftLang/SwiftEditor.cpp

```diff
@@ -330,7 +332,8 @@ bool SwiftEditorDocumentFileMap::getOrUpdate(
 
   bool found = false;
 
-  std::string ResolvedPath = SwiftLangSupport::resolvePathSymlinks(FilePath);
+  std::string ResolvedPath = resolveSymbolicLinks(FilePath,
+      llvm::vfs::getRealFileSystem().get());
```

Comment:
This shouldn't be the real filesystem, are we able to pass the FS down from the callers? The only place I would expect the real filesystem to be used is in sourcekitd-test.

Reply:
Found how to get the correct filesystem in here.